staging/velero/values.yaml

```diff
@@ -39,8 +39,8 @@ metrics:
   scrapeInterval: 30s
 
   service:
-    # labels:
-    #   key: value
+    labels:
```

Comment:
should this remain commented?

Reply:
Good question. I uncommented it to address a warning. The explanation is a [commit message](https://github.com/mesosphere/charts/pull/712/commits/fb99e536b34b8a4ae62c849415c450d1f8c6a431), so not easy to discover. I'll paste it here:

> Because the fields were commented in values.yaml, but referenced in the
> kubernetes-base-addons values, helm showed a warning:
>
>   `coalesce.go:199: warning: destination for service is a table. Ignoring non-table value <nil>`
>
>
> The warning refers to an _internal_, not-publicly-documented helm concept of a
> "YAML Table." See helm/helm#4514 for more
> information.
```